CQA Hammer CLI guide (#4987)
theforeman/foreman-documentation: guides/common/modules/proc_increasing-the-logging-level-for-hammer.adoc

= Increasing the logging level for Hammer

[role="_abstract"]

> Increase Hammer's logging level to debug for detailed troubleshooting information when diagnosing issues.

Reviewer: ```suggestion
Increase the logging level of Hammer to receive detailed troubleshooting information to debug issues.
```

Hammer's > of Hammer.
